lib/debugs_bunny/tasks/trace.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+namespace :debugs_bunny do
+  desc 'Delete expired trace records'
+  task delete_expired_traces: :environment do
+    max_age = DebugsBunny.configuration.max_age
+    DebugsBunny::Trace.older_than(max_age).destroy_all
```

Comment:
delete_all I think is a better choice. Are there any dependencies?

Reply:
There aren't any dependencies currently, but I used `destory_all` just in case dependencies get introduced, either on the DebugsBunny side or the client side. 
Is there a reason why `delete_all` is better?

Reply:
`delete_all` does everything in one sql statement, `destroy_all` does all the active record stuff, doing a quick test:

```
irb(main):002:0> Activity.destroy_all
  Activity Load (8.6ms)  SELECT "activities".* FROM "activities" /*application:Zetatango*/
   (1.8ms)  SAVEPOINT active_record_1 /*application:Zetatango*/
  Activity Destroy (1.5ms)  DELETE FROM "activities" WHERE "activities"."id" = $1 /*application:Zetatango*/  [["id", 203735561]]
   (0.2ms)  RELEASE SAVEPOINT active_record_1 /*application:Zetatango*/
   (0.2ms)  SAVEPOINT active_record_1 /*application:Zetatango*/
  Activity Destroy (0.4ms)  DELETE FROM "activities" WHERE "activities"."id" = $1 /*application:Zetatango*/  [["id", 100179845]]
   (0.2ms)  RELEASE SAVEPOINT active_record_1 /*application:Zetatango*/
   (0.2ms)  SAVEPOINT active_record_1 /*application:Zetatango*/
  Activity Destroy (0.4ms)  DELETE FROM "activities" WHERE "activities"."id" = $1 /*application:Zetatango*/  [["id", 841565714]]
   (0.2ms)  RELEASE SAVEPOINT active_record_1 /*application:Zetatango*/
   (0.2ms)  SAVEPOINT active_record_1 /*application:Zetatango*/
  Activity Destroy (0.3ms)  DELETE FROM "activities" WHERE "activities"."id" = $1 /*application:Zetatango*/  [["id", 768104492]]
   (0.2ms)  RELEASE SAVEPOINT active_record_1 /*application:Zetatango*/
```

```
irb(main):001:0> Activity.delete_all
   (0.3ms)  BEGIN /*application:Zetatango*/
  Activity Destroy (2.4ms)  DELETE FROM "activities" /*application:Zetatango*/
=> 4
```

Reply:
Good to know! Thanks for the info, I'll update the code